src/Aspire.ProjectTemplates/templates/aspire-empty/AspireApplication.1.ServiceDefaults/ServiceDefaultsExtensions.cs

```diff
@@ -3,20 +3,32 @@
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Diagnostics.HealthChecks;
 using Microsoft.Extensions.Logging;
-using Microsoft.Extensions.ServiceDiscovery;
+
 using OpenTelemetry;
 using OpenTelemetry.Metrics;
 using OpenTelemetry.Trace;
 
 namespace Microsoft.Extensions.Hosting;
 
-// Adds common .NET Aspire services: service discovery, resilience, health checks, and OpenTelemetry.
-// This project should be referenced by each service project in your solution.
-// To learn more about using this project, see https://aka.ms/dotnet/aspire/service-defaults
-public static class Extensions
+/// <summary>
+/// Adds common .NET Aspire services: service discovery, resilience, health checks, and OpenTelemetry.
+/// </summary>
+/// <remarks>
+/// This project should be referenced by each service project in your solution.
+/// To learn more about using this project, see https://aka.ms/dotnet/aspire/service-defaults
+/// </remarks>
+public static class ServiceDefaultsExtensions
 {
+    /// <summary>
+    /// Adds default services including service discovery, resilience, health checks, and OpenTelemetry to the specified <see cref="IHostApplicationBuilder"/>.
```

Comment:
The "service" in this case was actually supposed to refer to the kind of project this is being called from, i.e. these are the defaults for my services, where "services" means the applications I build and deploy, e.g. Customers Service, Basket Service, etc.

If we want to maintain that meaning, we should re-phrase this comment to something like:

```suggestion
    /// Adds and configures common concerns for service projects including service discovery, resilience, health checks, and OpenTelemetry to the specified <see cref="IHostApplicationBuilder"/>.
```